packages/table/src/cell/cell.tsx

```diff
@@ -102,7 +102,15 @@ export class Cell extends React.Component<ICellProps, {}> {
             },
         );
 
-        const content = <div className={textClasses}>{this.props.children}</div>;
+        const modifiedChildren = React.Children.map(this.props.children, (child) => {
```

Comment:
add comment somewhere explaining that `parentCell{Height,Width}` won't be used in the `Cell` except in `shouldComponentUpdate`